code/modules/mob/living/living.dm

```diff
@@ -1383,3 +1383,11 @@
 			STAMINA:<font size='1'><a href='?_src_=vars;[HrefToken()];mobToDamage=[refid];adjustDamage=stamina' id='stamina'>[getStaminaLoss()]</a>
 		</font>
 	"}
+
+/mob/living/proc/move_to_error_room()
+	var/obj/effect/landmark/error/error_landmark = locate(/obj/effect/landmark/error) in GLOB.landmarks_list
+	if(error_landmark)
+		forceMove(error_landmark.loc)
+	else
+		forceMove(locate(4,4,1)) //Even if the landmark is missing, this should put them in the error room.
```

Comment:
leave a comment saying you're sorry for whoever finds this when it inevitably breaks and also print a stack trace about the landmark missing

Reply:
Isn't a stack trace provided by `CRASH`? The comment is a good idear tho

Reply:
if erro's room has an area why not teleport to that

Reply:
I didn't see the crash for some reason